src/main.ts

```diff
@@ -0,0 +1,89 @@
+import "./styles/style.css";
+import { getData } from "./util";
+
+type responseType = { key: string; ascii: string; audio: string }[];
```

Comment:
Types must always begin with capital, and avoid Type suffix.  In this case `KeyInfo` would have been ideal

Reply:
Done!